packages/components/src/unit-control/test/index.tsx

```diff
@@ -135,12 +135,7 @@ describe( 'UnitControl', () => {
 		} );
 
 		it( 'should render label if single units', () => {
-			render(
-				<UnitControl
-					units={ [ { value: '%', label: '%' } ] }
-					value="30%"
```

Comment:
Removing `value` prop from test, which was added as a temporary fix in #40697 (see https://github.com/WordPress/gutenberg/pull/40697#discussion_r861178483 for context)